lib/cbuffer.h

```diff
@@ -40,6 +40,12 @@ extern size_t lib_cbufRead(cbuffer_t *buf, void *data, size_t sz);
 extern size_t lib_cbufWrite(cbuffer_t *buf, const void *data, size_t sz);
 
 
+extern int lib_cbufReadByte(cbuffer_t *buf, char *data) __attribute__((section(".noxip")));
```

Comment:
Other function prototypes does not have the attribute - is it needed or not (I think not)?

Reply:
Ack